Cargo.toml

```diff
@@ -57,7 +57,7 @@ iter-enum = "1"
 itertools = "0.10"
 pin-project = "1.0"
 serde = { version = "1", features = ["derive"] }
-sqlparser = { version = "0.10", features = ["serde"] }
+sqlparser = { git = "https://github.com/sqlparser-rs/sqlparser-rs/", rev = "014b82f03d82d6f36b5271376e33461002709a55" }
```

Comment:
Now we no more need `serde` in `sqlparser`, I wasn't aware of it.
Thanks that you removed it.